firmware/controllers/idle_thread.cpp

```diff
@@ -255,11 +268,12 @@ static msg_t ivThread(int param) {
 			lastCrankingCyclesCounter = engine->rpmCalculator.getRevolutionCounterSinceStart();
 			baseIdlePosition = iacPosition;
 		} else {
+			float manualIdlePosition = manualIdleController(cltCorrection);
 			if (engineConfiguration->idleMode == IM_MANUAL) {
 				// let's re-apply CLT correction
-				iacPosition = manualIdleController(cltCorrection);
+				iacPosition = manualIdlePosition;
 			} else {
-				iacPosition = autoIdle(cltCorrection);
+				iacPosition = autoIdle(cltCorrection, manualIdlePosition);
```

Comment:
I am a bit confused here. I would think that idle is either MANUAL or AUTO. With the change, it looks like manualIdleController(cltCorrection) is used in both MANUAL and AUTO modes, this is a bit unexpected?

Reply:
I suggest it only as a temporary solution. 
See https://rusefi.com/forum/viewtopic.php?p=27880#p27880